Classes/common/CDTMutableDocumentRevision.m

```diff
@@ -50,25 +49,14 @@ - (instancetype)initWithDocumentId:(NSString*) documentId
     
     if (self){
         _docId = documentId;
-        _private_body = body ? body : [NSMutableDictionary dictionary];
-        _private_attachments = attachments ? attachments : [NSMutableDictionary dictionary];
+        _body = body ? body : [NSMutableDictionary dictionary];
+        _attachments = attachments ? attachments : [NSMutableDictionary dictionary];
         _sourceRevId = sourceRevId;
     }
     
     return self;
 }
 
-- (void)setBody:(NSDictionary *)body { self.private_body = [body mutableCopy]; }
-
-- (NSMutableDictionary *)body { return self.private_body; }
-
-- (NSMutableDictionary *)attachments { return self.private_attachments; }
-
-- (void)setAttachments:(NSMutableDictionary *)attachments
-{
-    self.private_attachments = [attachments mutableCopy];
```

Comment:
although it changes the contract, add `copy` to the property results in errors since the calling `copy` on `NSMutableDictionary` results in an instance of `NSDictionary` so we can only store the pointer. There are a bunch of warnings in the tests, which I will go and clean up though.